src/latexify/codegen/function_codegen.py

```diff
@@ -425,7 +425,26 @@ def visit_Call(self, node: ast.Call) -> str:
 
         if special_latex is not None:
             return special_latex
-
+ 
+        # Special treatment for np.zeros
+        if func_name == "zeros":
+            str = ""
+            open_bracket = "{"
+            close_bracket = "}"
+            for i, elt in enumerate(node.args[0].elts):
+                str += "{" + self.visit(elt) + "}"
+                if i != len(node.args[0].elts) - 1:
+                    str += " " + r"\times" + " "
+
+            matrix_str = "0^" + open_bracket + str + close_bracket
+            return matrix_str
```

Comment:
General suggestions:

- Requires complete syntax checking.
- Don't use `str` as a variable as it is reserved by the builtin type name. Overwriting builtins will confuse the behavior of the code (even if the current code doesn't involve any errors, it will happen in the future). Use `latex` instead if you need to store some generated strings.
- Suppress string concatenation as [it is significantly expensive](https://stackoverflow.com/a/3055541). Making a sequence (generator), then join all of them is generally better.
- I think `\mathbf{0}` ( $\mathbf{0}$ ) should be used to distinguish that this is not a scalar $0$.

```suggestion
            if len(node.args) != 1:
                # fall back to the default.
                
            arg0 = node.args[0]
                
            if not isinstance(arg0, ast.Tuple):
                # fall back to the default.
                # Tecunically we don't support `zeros(n)` where `n` is a scalar.
            
            dims_latex = r" \times ".join(self.visit(x) for x in arg0.elts)
            return fr"\mathbf{{0}}^{{{dims_latex}}}"
```

Reply:
@odashi I made some changes to my code. 

- np.zeros and np.identity are now in a new function, treated as special numpy methods.
- I added some if statements checking for valid ast inputs. 
- I added some parametric tests. 

Some more considerations:

- Some string concatenation is simplified, some could be simplified further.
- In the case of np.zeros(0), I just represent it as $\mathbf{0}^{{1} \times {0}}$. There might be better ways of expressing it as a special case. Do you have any suggestions?

Let me know what you think.